Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/SettingsFile.java

```diff
@@ -321,9 +320,11 @@ public static void saveFile(final String fileName, final HashMap<String, Setting
 		{
 			writer = new PrintWriter(ini, "UTF-8");
 
+			//Add Sorting for output, change keySet into sortKeySet and itterate through the new sorted set
```

Comment:
There should be a space after the `//`. The comment could use some rewording too (or maybe it can be removed altogether, since the name `sortedKeySet` makes it clear that we're doing this because we want it to be sorted?)